src/Drivers/LaravelZeroDriver.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use LaravelZero\Framework\Application;
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\Submenu;
+use Tinkerwell\ContextMenu\SetCode;
+use Tinkerwell\ContextMenu\OpenURL;
+
+class LaravelZeroTinkerwellDriver extends TinkerwellDriver
+{
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/bootstrap/app.php')
+            && is_dir($projectPath . '/vendor/laravel-zero/framework');
```

Comment:
Just wondering if a `class_exists()` would be better here? But not really sure.
```suggestion
            && class_exists(Application:class);
```

Reply:
That does look cleaner :)

Reply:
@owenvoke I'll close this PR so you can create your own so the code is on your name!

Reply:
Ah, sorry. That's not quite what I meant. 😬 But if that's easier, I'll create a PR. 👍

And actually, just realised we can't use a `class_exists()` because the class won't have been loaded.